i2c/oled_i2c/oled_i2c.c

```diff
@@ -0,0 +1,301 @@
+/**
+ * Copyright (c) 2021 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <stdio.h>
+#include "pico/stdlib.h"
+#include "pico/binary_info.h"
+#include "hardware/i2c.h"
+#include "raspberry26x32.h"
+
+ /* Example code to talk to an SSD1306-based OLED display
+
+    NOTE: Ensure the device is capable of being driven at 3.3v NOT 5v. The Pico
+    GPIO (and therefore I2C) cannot be used at 5v.
+
+    You will need to use a level shifter on the I2C lines if you want to run the
+    board at 5v.
+
+    Connections on Raspberry Pi Pico board, other boards may vary.
+
+    GPIO PICO_DEFAULT_I2C_SDA_PIN (on Pico this is 4 (pin 6)) -> SDA on display
```

Comment:
We tend to refer to "GP4" rather than just use "4", and then "physical pin 6" if we referring to the physical numbering of pins. So that'd be "on Raspberry Pi Pico this is GP4 (physical pin 6)".

Reply:
I think this comment also applies to your BMP280 PR?

Reply:
Agreed..fixed both.